channels/ca.m3u

```diff
@@ -165,6 +167,8 @@ https://live.corusdigitaldev.com/groupd/live/deb8cec5-87fb-460a-ab82-0929374fc5f
 https://live.corusdigitaldev.com/groupb/live/564df695-94f9-4f27-b1b9-0a936ab01721/live.isml/.m3u8
 #EXTINF:-1 tvg-id="GurSikhSabhaTV.ca" tvg-country="CA" tvg-language="Hindi" tvg-logo="https://i.imgur.com/2EKCtQm.png" group-title="Religious",GurSikh Sabha TV (720p)
 http://cdn12.henico.net:8080/live/gsctv/index.m3u8
+#EXTINF:-1 tvg-id="RamgarhiABC.com" tvg-country="CA" tvg-language="Punjabi" tvg-logo="http://ramgarhiabc.com/wp-content/uploads/2014/02/headerlogo.png" group-title="",Ramgarhi ABC
```

Comment:
```suggestion
#EXTINF:-1 tvg-id="RamgarhiABC.ca" tvg-country="CA" tvg-language="Punjabi" tvg-logo="http://ramgarhiabc.com/wp-content/uploads/2014/02/headerlogo.png" group-title="",Ramgarhi ABC
```

Reply:
@freearhey Can we use site address in the tvg-id? Some channels are located at standard `.com` address and do not end with `.ca`.